apps/frontend/src/pages/project-access-management/libs/components/project-group-create-form/libs/helpers/get-users-from-project-groups.helper.ts

```diff
@@ -0,0 +1,34 @@
+import { type ProjectGroupGetAllItemResponseDto } from "~/modules/project-groups/project-groups.js";
+import { type UserGetAllItemResponseDto } from "~/modules/users/users.js";
+
+const getUsersFromProjectGroups = (
+	projectGroups: ProjectGroupGetAllItemResponseDto[],
+): Omit<UserGetAllItemResponseDto, "email">[] => {
+	const userProjectGroupsMap: Record<
+		number,
+		Omit<UserGetAllItemResponseDto, "email">
+	> = {};
+
+	for (const projectGroup of projectGroups) {
+		for (const user of projectGroup.users) {
+			const userGroups = userProjectGroupsMap[user.id]?.groups ?? [];
+
+			userProjectGroupsMap[user.id] =
+				(user.id,
+				{
+					...user,
+					groups: [
+						...userGroups,
+						{
+							id: projectGroup.id,
+							name: projectGroup.name,
+						},
+					],
+				});
```

Comment:
The comma operator is used here, but it only evaluates user.id and then discards it, which isn't necessary. It doesn't break the code, but it can be confusing.